.env.test

```diff
@@ -9,4 +9,6 @@ SECRET_KEY=secretkey
 ENDPOINT=mongodb://fakemongo.com
 DATABASE=mh-todoapp-2022
 
-LOGIN_DISABLED=True
\ No newline at end of file
+LOGIN_DISABLED=True
+LOG_LEVEL=DEBUG
+LOGGLY_TOKEN=testtoken
```

Comment:
You may want to add this setting to the other .env files, so that you can test/perform logging manually

Reply:
Make sure LOG_LEVEL is added to the `.env.template` as well